### ISSUE TITLE
Could not find type 'RatScanner.FetchModels.TarkovDev.TaskObjectiveTraderStanding' in assembly 'RatScanner, Version=3.6.5.0, Culture=neutral, PublicKeyToken=null'.

### ISSUE THREAD

**Issue by JanK118** [RatScanner/RatScanner#652]
Hi, 

I got this message today when opening RatScanner. Everything was working normal 24 hours ago, nothing changed till yesterday.

I tried running it as administrator (which I didn't do before) and restarted my workstation.

I am running Windows 11 x64.

**Error**
Could not find type 'RatScanner.FetchModels.TarkovDev.TaskObjectiveTraderStanding' in assembly 'RatScanner, Version=3.6.5.0, Culture=neutral, PublicKeyToken=null'.
```
System.Reflection.TargetInvocationException: Exception has been thrown by the target of an invocation.
 ---> Newtonsoft.Json.JsonSerializationException: Error resolving type specified in JSON 'RatScanner.FetchModels.TarkovDev.TaskObjectiveTraderStanding, RatScanner'. Path 'data.tasks[355].objectives[0].$type', line 1, position 606363.
 ---> Newtonsoft.Json.JsonSerializationException: Could not find type 'RatScanner.FetchModels.TarkovDev.TaskObjectiveTraderStanding' in assembly 'RatScanner, Version=3.6.5.0, Culture=neutral, PublicKeyToken=null'.
   at Newtonsoft.Json.Serialization.DefaultSerializationBinder.GetTypeFromTypeNameKey(StructMultiKey`2 typeNameKey)
   at System.Collections.Concurrent.ConcurrentDictionary`2.GetOrAdd(TKey key, Func`2 valueFactory)
   at Newtonsoft.Json.Utilities.ThreadSafeStore`2.Get(TKey key)
   at Newtonsoft.Json.Serialization.DefaultSerializationBinder.GetTypeByName(StructMultiKey`2 typeNameKey)
   at Newtonsoft.Json.Serialization.DefaultSerialization
```
<details>
<summary>Log</summary>

```
[23:39:05.9344908] > [Info]  ----- RatScanner 3.6.5 -----
[23:39:05.9346417] > [Info]  Screen Info: 2560x1440 at 100%
[23:39:05.9346452] > [Info]  Setting temporary default item...
[23:39:05.9351524] > [Info]  Initializing tarkov tracker database
[23:39:05.9352288] > [Info]  Initializing hotkey manager...
[23:39:05.9375701] > [Info]  UI Ready!
[23:39:06.4880595] > [Error] Could not find type 'RatScanner.FetchModels.TarkovDev.TaskObjectiveTraderStanding' in assembly 'RatScanner, Version=3.6.5.0, Culture=neutral, PublicKeyToken=null'.
 -------------------- 
 System.Reflection.TargetInvocationException: Exception has been thrown by the target of an invocation.
 ---> Newtonsoft.Json.JsonSerializationException: Error resolving type specified in JSON 'RatScanner.FetchModels.TarkovDev.TaskObjectiveTraderStanding, RatScanner'. Path 'data.tasks[355].objectives[0].$type', line 1, position 606363.
 ---> Newtonsoft.Json.JsonSerializationException: Could not find type 'RatScanner.FetchModels.TarkovDev.TaskObjectiveTraderStanding' in assembly 'RatScanner, Version=3.6.5.0, Culture=neutral, PublicKeyToken=null'.
   at Newtonsoft.Json.Serialization.DefaultSerializationBinder.GetTypeFromTypeNameKey(StructMultiKey`2 typeNameKey)
   at System.Collections.Concurrent.ConcurrentDictionary`2.GetOrAdd(TKey key, Func`2 valueFactory)
   at Newtonsoft.Json.Utilities.ThreadSafeStore`2.Get(TKey key)
   at Newtonsoft.Json.Serialization.DefaultSerializationBinder.GetTypeByName(StructMultiKey`2 typeNameKey)
   at Newtonsoft.Json.Serialization.DefaultSerializationBinder.BindToType(String assemblyName, String typeName)
   at Newtonsoft.Json.Serialization.JsonSerializerInternalReader.ResolveTypeName(JsonReader reader, Type& objectType, JsonContract& contract, JsonProperty member, JsonContainerContract containerContract, JsonProperty containerMember, String qualifiedTypeName)
   --- End of inner exception stack trace ---
   at Newtonsoft.Json.Serialization.JsonSerializerInternalReader.ResolveTypeName(JsonReader reader, Type& objectType, JsonContract& contract, JsonProperty member, JsonContainerContract containerContract, JsonProperty containerMember, String qualifiedTypeName)
   at Newtonsoft.Json.Serialization.JsonSerializerInternalReader.ReadMetadataProperties(JsonReader reader, Type& objectType, JsonContract& contract, JsonProperty member, JsonContainerContract containerContract, JsonProperty containerMember, Object existingValue, Object& newValue, String& id)
   at Newtonsoft.Json.Serialization.JsonSerializerInternalReader.CreateObject(JsonReader reader, Type objectType, JsonContract contract, JsonProperty member, JsonContainerContract containerContract, JsonProperty containerMember, Object existingValue)
   at Newtonsoft.Json.Serialization.JsonSerializerInternalReader.CreateValueInternal(JsonReader reader, Type objectType, JsonContract contract, JsonProperty member, JsonContainerContract containerContract, JsonProperty containerMember, Object existingValue)
   at 
```
</details>

**Comment by JanK118** [comment]
Ah, sorry, didn't see the one which was tagged as "closed".

**Comment by Blightbuster** [comment]
Hey, thanks for reporting was just fixed in v3.6.6.
Just start Rat Scanner and it will ask you to update.
